docker-hub/vulnerability-scanning.md

```diff
@@ -1,124 +1,174 @@
+keywords: scanning, vulnerabilities, Hub, basic
+title: Basic vulnerability scanning
 ---
 
-{% include upgrade-cta.html
-  body="Docker Hub Vulnerability Scanning is available for users subscribed to Docker Pro, Team, or a Business subscription. Upgrade now to get automatic vulnerability scans for your images."
-  header-text="This feature requires a paid Docker subscription"
-  target-url="https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade_scan"
-%}
+Docker Hub vulnerability scanning lets you automatically scan Docker images for
+vulnerabilities.
 
-Docker Hub Vulnerability Scanning enables you to automatically scan Docker
-images for vulnerabilities using Snyk. This uses the same technology as the
-[docker scan](../engine/scan/index.md) command.
+When you push an image to a Docker Hub repository after turning on vulnerability
+scanning, Docker Hub automatically scans the image to identify vulnerabilities.
+Vulnerability Scanning lets you review the security state of your images and
+take actions to fix issues identified during the scan, resulting in more secure
+deployments.
 
-> When you enable Hub Vulnerability Scanning, you can also see whether your
-> images are affected by Log4Shell (CVE-2021-44228). For more information, see
-> [Scan images](#scan-images-on-docker-hub).
+Scan results include:
 
-When you push an image to Docker Hub after enabling vulnerability scanning, Docker Hub automatically scans the image to identify vulnerabilities in your container images. Vulnerability Scanning allows developers and development teams to review the security state of the container images and take actions to fix issues identified during the scan, resulting in more secure deployments. The scan result includes the source of the vulnerability, such as OS packages and libraries, version in which it was introduced, and a recommended fixed version (if available) to remediate the vulnerabilities discovered.
+- The source of the vulnerability, such as OS packages and libraries
+- The version in which it was introduced
+- A recommended fixed version (if available) to remediate the vulnerabilities
+  discovered.
 
-## Scan images on Docker Hub
+## Changes to vulnerability scanning in Docker Hub
 
-> **Log4j 2 CVE-2021-44228**
->
-> Docker Hub security scans triggered after **1700 UTC 13 December 2021** are
-> now correctly identifying the Log4j2 CVE. Scans before this date **do not**
-> currently reflect the status of this vulnerability. Therefore, we recommend
-> that you trigger scans by pushing the image to Docker Hub to view the status
-> of Log4j2 CVE in the vulnerability report.
-{: .important}
+As of February 23rd 2023, Docker is changing the technology that supports
+vulnerability scanning Docker Hub. Vulnerability scanning now uses
+[Docker Scout](../scout/index.md) to detect vulnerabilities.
+
+As a result of this change, vulnerability scanning now detects a higher number
```

Comment:
I think the bit we want to communicate is that Hub vulnerability scanning now looks in more places, so will likely identify more vulnerabilities (at the moment it reads like it's going to pick up more vulnerabilities alone)